app/mobile-content/media/media-gallery/media-gallery.component.ts

```diff
@@ -114,96 +114,87 @@ export class MediaGalleryComponent implements OnInit {
       (obj: any) => {
         let editedMedium: Medium = obj.media;
         let file: File = obj.file;
-        if (editedMedium) {
-          this.mediaService.updateMedia(editedMedium)
-            .then(
-              (res: any) => {
-                this.toasterService.pop('success', this.translate('media updated'));
-                medium.description = editedMedium.description;
-                medium.type = editedMedium.type;
-                medium.status = editedMedium.status;
-                medium.title = editedMedium.title;
-                if (file) {
-                  setTimeout(
-                    () => {
-                    return this.mediaService.uploadFile(editedMedium.id, file);
-                  }, 3000);
-                }
-              }
-            ).catch(
-              (err) => {
-                this.toasterService.pop('error', this.translate('Error while updating'), err);
+        if (!editedMedium) { return; }
```

Comment:
I like the style that was used before better. For the readability of the code it is better IMO to have the whole block inside an `if` instead of using the negated condition to `return` prematurely.  Can you change that back? Or do I misunderstand something here?

Reply:
Hm, I would dispute that. If the code is sufficiently complex, you'll end up with lots of brackets and indentation levels, which I personally find less readable than the pretty succinct return statement. I only changed it because I didn't like the giant layer cake of brackets. But ok, I'll change it.